invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/globals/site.overrides

```diff
@@ -547,6 +551,10 @@ dl.details-list {
     align-items: flex-start;
     row-gap: 1rem;
   }
+
+  &.three-columns {
```

Comment:
this can be avoided by nesting divs, instead of adding a new class to handle 3 columns